packages/server/src/plugin/usageReporting/plugin.ts

```diff
@@ -677,10 +658,10 @@ export function ApolloServerPluginUsageReporting<TContext extends BaseContext>(
                 statsReportKey,
                 trace,
                 // We include the operation as a trace (rather than aggregated
-                // into stats) only if we believe it's possible that our
-                // organization's plan allows for viewing traces *and* we
-                // actually captured this as a full trace *and*
-                // sendOperationAsTrace says so.
+                // into stats) only if the user didn't set `sendTraces: false`
+                // *and8 we believe it's possible that our organization's plan
```

Comment:
typo `*and8`

Reply:
@bonnici caught this and fixed it elsewhere